src/System.Threading/tests/BarrierTests.cs

```diff
@@ -263,7 +263,7 @@ public static void RunBarrierTest8_PostPhaseException()
                     if (shouldThrow)
                         throw new InvalidOperationException();
                 });
-            int succeededCount = 0;
+            long succeededCount = 0;
```

Comment:
Why did this change?  If this is a way of getting coverage for Interlocked.Increment, that's very subtle, and I'd suggest we have dedicated tests for it.